easybuild/easyconfigs/v/Virtuoso-opensource/Virtuoso-opensource-7.2.6.1-foss-2021a.eb

```diff
@@ -0,0 +1,71 @@
+##
+# This is a contribution from SIB Swiss Institute of Bioinformatics
+# Homepage:     https://www.sib.swiss/research-infrastructure/competence-centers/vital-it
+#
+# Authors::     Sebastien Moretti <sebastien.moretti@sib.swiss>
+#
+##
+easyblock = 'ConfigureMake'
+
+name = 'Virtuoso-opensource'
+version = '7.2.6.1'
+
+homepage = 'https://github.com/openlink/virtuoso-opensource'
+description = """Virtuoso is a high-performance and scalable Multi-Model RDBMS, Data
+Integration Middleware, Linked Data Deployment, and HTTP Application Server Platform."""
+software_license = 'LicenseGPLv2'
+
+toolchain = {'name': 'foss', 'version': '2021a'}
```

Comment:
Does `Virtuoso` actually require MPI or BLAS/LAPACK/FFTW? If not, change this to the `GCC` toolchain?

Reply:
No, so moved to `GCC` toolchain